tests/server/api/v1/test_responses.py

```diff
@@ -171,6 +173,58 @@ def test_update_response_with_nonexistent_response_id(client: TestClient, db: Se
     }
 
 
+def test_update_response_status(client: TestClient, db: Session, admin_auth_header: dict):
+    response = ResponseFactory.create()
+    response_json = {"status": "submitted"}
+
+    resp = client.put(f"/api/v1/responses/{response.id}/status", headers=admin_auth_header, json=response_json)
+
+    assert resp.status_code == 200
+    assert db.get(Response, response.id).status == "submitted"
+
+
```

Comment:
It would be nice to check the request body response in the same way as `test_update_response`.